src/panels/lovelace/cards/hui-glance-card.ts

```diff
@@ -67,7 +67,7 @@ export class HuiGlanceCard extends hassLocalizeLitMixin(LitElement)
 
   public setConfig(config: Config): void {
     this._config = { theme: "default", ...config };
-    const entities = processConfigEntities(config.entities);
+    const entities = processConfigEntities(config.entities) as ConfigEntity[];
```

Comment:
We should have `processConfigEntities` take a type that you expect return values to be. 

(also, `ConfigEntity` and `EntityConfig`, we must really hate our future selves picking such confusing names)

Reply:
```suggestion
    const entities = processConfigEntities<ConfigEntity>(config.entities);
```

Reply:
Yeah, reworking types for cards is on my todo list